src/protocolsupport/protocol/packet/middleimpl/serverbound/play/v_pe/Interact.java

```diff
@@ -35,7 +38,7 @@ public void readFromClientData(ByteBuf clientdata) {
 		switch (peAction) {
 			case INTERACT: {
 				if ((target != null) && (target.getType() != NetworkEntityType.ARMOR_STAND)) {
```

Comment:
Actually need to check what values are sent when client is interating not with armor stand. Maybe some magic values are use, which could allow us to not use the cache